packages/core/src/mpi/match-patients.ts

```diff
@@ -157,17 +157,37 @@ export function exactMatchSimilarity(patient1: PatientData, patient2: PatientDat
 }
 
 function isSameContactByPhone(a?: Contact, b?: Contact): boolean {
-  return a?.phone === b?.phone;
+  return (
+    a?.phone !== undefined &&
+    a.phone !== null &&
+    b?.phone !== undefined &&
+    b.phone !== null &&
```

Comment:
I think this can be simplified to:

if !!a?.phone && !!b?.phone

https://stackoverflow.com/questions/7452720/what-does-the-double-exclamation-operator-mean

Reply:
good call

Reply:
done